### ISSUE TITLE
FR - Colorized Tags and Account Names

### ISSUE THREAD

**Issue by Titus1024** [nuxsmin/sysPass#1077]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
The current color configuration of the Tags is hard to read as they are on a white background, surrounded by a light grey box and then white text with no border on the text.
![image](https://user-images.githubusercontent.com/22060361/47167189-89f83180-d2c3-11e8-97fb-3473dfc2c367.png)
See above for an example.
If there is a way to change this I have not been able to find it.
Having the ability to change this would be optimal, or at least have the current color configuration be more readable.

I noticed that the color in the circle before the account name changes on login, which appears to be random and I have not been able to find a way to control this.
![image](https://user-images.githubusercontent.com/22060361/47167328-d93e6200-d2c3-11e8-9be0-edcfb2ffd07c.png)
See above for an example.
Being able to control this would be great as well, as using a color for association to a specific account is quite helpful, in my opinion at least.

**Describe alternatives you've considered**
None that I know of as I have not been able to find any options in the UI to change this.

**Comment by nuxsmin** [comment]
Hello, some theme behavior/appearance can be modified by cloning the current theme an then modify the related resources (ie. views, CSS or JavaScript). The tags opacity is modified to get a distraction-free interface and keep it focused on the accounts data.

Regarding the customer/client colours, they're assigned in a random way, but keeping the visual harmony on the material design palette. Hopefully they will be able to be changed per client on v3.

Regards

**Comment by Titus1024** [comment]
Interesting, distraction free I kind of understand but I feel like straining to see what the tag says seems a bit impractical.

**Comment by nuxsmin** [comment]
I agree, it might be improved a bit and honestly my first thought about the current implementation is that tags are mainly used for filtering, so the tags box would be most useful (and used) than selecting by clicking over the account's row.

Anyway, it's a good point of view ;)
